src/routes/transactions/mappers/common/transaction-info.mapper.ts

```diff
@@ -131,7 +131,7 @@ export class MultisigTransactionInfoMapper {
         if (error instanceof NotFoundException) {
           this.loggingService.warn(error);
         } else {
-          throw error;
+          this.loggingService.error(error);
```

Comment:
Ok, but what are we then mapping below if we have an exception here? why is the code afterwards not throwing exceptions?